apps/web/src/pages/CommunityInterests/sections/TrainingAndDevelopmentOpportunities.tsx

```diff
@@ -0,0 +1,285 @@
+          eligibleClassifications {
+            id
+            group
+            level
+          }
+        }
+      }
+    }
+  `,
+);
+
+export interface SubformValues {
+  interestInDevelopmentPrograms:
+    | {
+        developmentProgramId: string | null;
+        participationStatus: string | null;
+        completionDate: string | null;
+      }[]
+    | null;
+}
+
+interface TrainingAndDevelopmentOpportunitiesProps {
+  optionsQuery: FragmentType<
+    typeof TrainingAndDevelopmentOpportunitiesOptions_Fragment
+  >;
+  formDisabled: boolean;
+}
+
+const TrainingAndDevelopmentOpportunities = ({
+  optionsQuery,
+  formDisabled,
+}: TrainingAndDevelopmentOpportunitiesProps) => {
+  const intl = useIntl();
+  const optionsData = getFragment(
+    TrainingAndDevelopmentOpportunitiesOptions_Fragment,
+    optionsQuery,
+  );
+
+  const { watch, register } = useFormContext<FormValues>();
+  const [selectedCommunityId, selectedInterestInDevelopmentPrograms] = watch([
+    "communityId",
+    "interestInDevelopmentPrograms",
+  ]);
+
+  const developmentPrograms =
+    optionsData.communities.find(
+      (community) => community?.id === selectedCommunityId,
+    )?.developmentPrograms ?? [];
+
+  // sort programs
+  developmentPrograms.sort((a, b) =>
+    (a.name?.localized ?? "").localeCompare(b.name?.localized ?? ""),
+  );
+  // sort classifications in programs
+  developmentPrograms.forEach((developmentProgram) => {
+    if (developmentProgram.eligibleClassifications) {
+      developmentProgram.eligibleClassifications.sort((a, b) =>
+        getClassificationName(a, intl).localeCompare(
+          getClassificationName(b, intl),
+        ),
+      );
+    }
+  });
+
+  return (
+    <div
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(column)"
+      data-h2-gap="base(x1.25)"
+    >
+      {/* heading and description */}
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <Heading
+          level="h2"
+          data-h2-font-weight="base(400)"
+          Icon={RectangleGroupIcon}
+          color="primary"
+          data-h2-margin="base(0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Training and development opportunities",
+            id: "I5K1zI",
+            description:
+              "Heading for the 'Training and development opportunities' section",
+          })}
+        </Heading>
+        <span>
+          {intl.formatMessage({
+            defaultMessage:
+              "Most functional communities offer various programs for learning and development. These programs might have specific eligibility requirements based on your experience, classification, or other qualifications. Expressing interest in these opportunities isn’t an application, but it allows HR and recruitment staff to verify your interest in case you’ve been nominated for a training or development opportunity.",
```

Comment:
Avoid abbreviations for accessibility purposes.

```suggestion
              "Most functional communities offer various programs for learning and development. These programs might have specific eligibility requirements based on your experience, classification, or other qualifications. Expressing interest in these opportunities isn’t an application, but it allows human resources and recruitment staff to verify your interest in case you’ve been nominated for a training or development opportunity.",
```